src/api/stats/avax/getPangolinV2DualApys.js

```diff
@@ -0,0 +1,145 @@
+const BigNumber = require('bignumber.js');
+const { MultiCall } = require('eth-multicall');
+const { avaxWeb3: web3, multicallAddress } = require('../../../utils/web3');
+
+const MasterChef = require('../../../abis/avax/PangolinChef.json');
+const Rewarder = require('../../../abis/avax/PangolinRewarderViaMultiplier.json');
+const ERC20 = require('../../../abis/ERC20.json');
+const fetchPrice = require('../../../utils/fetchPrice');
+const pools = require('../../../data/avax/pangolinV2DualLpPools.json');
+const { BASE_HPY, AVAX_CHAIN_ID } = require('../../../constants');
+const { getTradingFeeAprSushi } = require('../../../utils/getTradingFeeApr');
+import { getFarmWithTradingFeesApy } from '../../../utils/getFarmWithTradingFeesApy';
+const { pangolinClient } = require('../../../apollo/client');
+const { compound } = require('../../../utils/compound');
+
+const masterchef = '0x1f806f7C8dED893fd3caE279191ad7Aa3798E928';
+const oracleIdA = 'PNG';
+const oracleA = 'tokens';
+const DECIMALSA = '1e18';
+
+const secondsPerBlock = 1;
+const secondsPerYear = 31536000;
+
+const liquidityProviderFee = 0.0025;
+const beefyPerformanceFee = 0.045;
+const shareAfterBeefyPerformanceFee = 1 - beefyPerformanceFee;
+
+const getPangolinV2DualApys = async () => {
+  let apys = {};
+  let apyBreakdowns = {};
+
+  const tokenPriceA = await fetchPrice({ oracle: oracleA, id: oracleIdA });
+  const { rewardPerSecond, totalAllocPoint } = await getMasterChefData();
+  const { balances, allocPoints, rewarders } = await getPoolsData(pools);
+
+  const pairAddresses = pools.map(pool => pool.address);
+  const tradingAprs = await getTradingFeeAprSushi(
```

Comment:
tradingAprs aren't calculated, but if you change it to just `getTradingFeeApr` it will be ok